pgjdbc/src/main/java/org/postgresql/core/NativeQuery.java

```diff
@@ -27,13 +28,14 @@
   }
 
   public NativeQuery(String nativeSql) {
-    this(nativeSql, NO_BINDS);
+    this(nativeSql, NO_BINDS, false);
   }
 
-  public NativeQuery(String nativeSql, int[] bindPositions) {
+  public NativeQuery(String nativeSql, int[] bindPositions, boolean compatible) {
```

Comment:
@polobo sorry, have just seen your comment shortly after pushing a commit. I agree the parameter name can be improved. I'll push out another commit shortly to re-name it.